package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "local-home-virual-environment",
+  "name": "local-home-virtual-environment",
   "version": "0.0.1",
```

Comment:
bump/clean dependencies?
- more recent version of ava and typescript?
- use https://github.com/google/gts
- is webpack needed?

Reply:
Webpack not needed.  I'll fix the dependencies and update this PR description to reflect the dependency changes.

Reply:
I cleaned the dependencies up and applied `gts`